### PR TITLE
[Cleaning] includes, nullptr, default, dtk->qt, error dbcontroller

### PR DESCRIPTION
**Pull request by mathildemerle**
While cleaning some files about `includes`, `NULL` -> `nullptr`, `=default`, and dtk logs -> qt logs, i found an error in 
`medDatabaseController::createSeriesTable()`. There were no  `return` at the complete end of the function.

:m:

### CODE REVIEW

File: src/layers/legacy/medPacs/medAbstractPacsBaseScu.h [hunk 1]
```diff
@@ -24,8 +24,8 @@ class MEDPACS_EXPORT medAbstractPacsBaseScu : public QObject
     Q_OBJECT
     
 public:
-             medAbstractPacsBaseScu();
-    virtual ~medAbstractPacsBaseScu();
+    medAbstractPacsBaseScu() = default;
+    ~medAbstractPacsBaseScu() = default;
```
Comment (Florent2305) [inline src/layers/legacy/medPacs/medAbstractPacsBaseScu.h]: same as previous

File: src/layers/legacy/medPacs/medAbstractPacsEchoScu.h [hunk 2]
```diff
@@ -21,12 +21,9 @@ class MEDPACS_EXPORT medAbstractPacsEchoScu : public medAbstractPacsBaseScu
     Q_OBJECT
 
 public:
-             medAbstractPacsEchoScu();
-    virtual ~medAbstractPacsEchoScu();
+    medAbstractPacsEchoScu() = default;
+    ~medAbstractPacsEchoScu() = default;
```
Comment (Florent2305) [inline src/layers/legacy/medPacs/medAbstractPacsEchoScu.h]: same as previous

File: src/layers/legacy/medPacs/medAbstractPacsFindScu.h [hunk 3]
```diff
@@ -21,11 +21,8 @@ class MEDPACS_EXPORT medAbstractPacsFindScu : public medAbstractPacsBaseScu
     Q_OBJECT
 
 public:
-             medAbstractPacsFindScu();
-    virtual ~medAbstractPacsFindScu();
+    medAbstractPacsFindScu() = default;
+    ~medAbstractPacsFindScu() = default;
```
Comment (Florent2305) [inline src/layers/legacy/medPacs/medAbstractPacsFindScu.h]: same as previous

File: src/layers/legacy/medPacs/medAbstractPacsMoveScu.h [hunk 4]
```diff
@@ -23,8 +23,8 @@ class MEDPACS_EXPORT medAbstractPacsMoveScu : public medAbstractPacsBaseScu
     Q_OBJECT
 
 public:
-             medAbstractPacsMoveScu();
-    virtual ~medAbstractPacsMoveScu();
+    medAbstractPacsMoveScu() = default;
+    ~medAbstractPacsMoveScu() = default;
```
Comment (Florent2305) [inline src/layers/legacy/medPacs/medAbstractPacsMoveScu.h]: same as previous

File: src/layers/legacy/medPacs/medAbstractPacsNode.h [hunk 5]
```diff
@@ -24,8 +24,8 @@ class MEDPACS_EXPORT medAbstractPacsNode : public QObject
     Q_OBJECT
     
 public:
-             medAbstractPacsNode();
-    virtual ~medAbstractPacsNode();
+    medAbstractPacsNode() = default;
+    ~medAbstractPacsNode() = default;
```
Comment (Florent2305) [inline src/layers/legacy/medPacs/medAbstractPacsNode.h]: same as previous

File: src/layers/legacy/medPacs/medAbstractPacsResultDataset.h [hunk 6]
```diff
@@ -24,8 +24,8 @@ class MEDPACS_EXPORT medAbstractPacsResultDataset : public QObject
     Q_OBJECT
 
 public:
-             medAbstractPacsResultDataset();
-    virtual ~medAbstractPacsResultDataset();
+    medAbstractPacsResultDataset() = default;
+    ~medAbstractPacsResultDataset() = default;
```
Comment (Florent2305) [inline src/layers/legacy/medPacs/medAbstractPacsResultDataset.h]: same as previous

File: src/layers/legacy/medPacs/medAbstractPacsStoreScp.h [hunk 7]
```diff
@@ -21,8 +21,8 @@ class MEDPACS_EXPORT medAbstractPacsStoreScp : public QObject
     Q_OBJECT
 
 public:
-    medAbstractPacsStoreScp();
-    virtual ~medAbstractPacsStoreScp();
+    medAbstractPacsStoreScp() = default;
+    ~medAbstractPacsStoreScp() = default;
```
Comment (Florent2305) [inline src/layers/legacy/medPacs/medAbstractPacsStoreScp.h]: same as previous

File: src/layers/legacy/medPacs/medAbstractPacsStoreScu.h [hunk 8]
```diff
@@ -22,11 +22,8 @@ class MEDPACS_EXPORT medAbstractPacsStoreScu : public medAbstractPacsBaseScu
     Q_OBJECT
 
 public:
-    medAbstractPacsStoreScu();
-    virtual ~medAbstractPacsStoreScu();
+    medAbstractPacsStoreScu() = default;
+    ~medAbstractPacsStoreScu() = default;
```
Comment (Florent2305) [inline src/layers/legacy/medPacs/medAbstractPacsStoreScu.h]: same as previous

**Comment by mathildemerle** [comment]
Done, @Florent2305  :)

**Pull request merged**